libs/platform/src/lib/components/panel/panel.config.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+import { PlatformConfig } from '../../platform.config';
+import { ContentDensity } from '../form/form-control';
+
+/**
+ * Default options for platform panel
+ */
+@Injectable({ providedIn: 'root' })
+export class PanelConfig {
+    /**
+     * ARIA label for button when the Panel is collapsed
+     */
+    expandLabel = 'Expand Panel';
+
+    /**
+     * ARIA label for button when the Panel is expanded
+     */
+    collapseLabel = 'Collapse Panel';
```

Comment:
Note: We are going to need a way to internationalize these default strings. We should not address this in current PR though, I think this is OK for now.